### ISSUE TITLE
Some icon images are missing in the tutorial

### ISSUE THREAD

**Issue by evertrol** [NLeSC/spot#182]
A few icon images are missing; the tutorial shows the typical missing image picture, and (on Chrome 69) the alt text instead. Since the larger images and some other icons are there, I assume this is something in the HTML / text itself, not because of GitHub pages.
In fact, inspecting the visible icons indicates there is a missing "images/" subdirectory missing for the image source.

Missing icons:

- Step 1, bullet list, item 3, end of second sentence: settings icon (gear wheel).
  (Side note: this icon is linked, but ends up in 404 itself: csv_options.jpg)

- Edit your data, end of first sentence: settings icon (gear wheel)

- Customizing plots, end of second sentence: settings icon (gear wheel)

- Scatter plot, second paragraph directly after the list, mid-way first sentence: settings icon (gear wheel)

**Comment by evertrol** [comment]
Moved to https://github.com/NLeSC/spot-tutorial/issues/8
